.changeset/cold-numbers-push/changes.md

```diff
@@ -0,0 +1 @@
+Renamed `passport-auth` to `auth-passport`
```

Comment:
Can we add a bit more details here, as this is actually a pretty significant change, and we don't want people to miss it. I'm thinking something like:

```
This change renames the package `@keystone-alpha/passport-auth` to `@keystone-alpha/auth-passport`. Anyone using `passport-auth` should switch over to the new package name, as the old one will no longer be receiving any updates.
```